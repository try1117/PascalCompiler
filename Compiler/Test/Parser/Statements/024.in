program test;
var
  i, a: integer;
  j: char;
begin
  for i := 1 to 10 begin
    j := j + 1;
  end;
end.
