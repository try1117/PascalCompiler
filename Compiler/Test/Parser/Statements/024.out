Syntax exception in position (6,20) - Expected "Do" but found "begin"
